src/config/env.ts

```diff
@@ -40,11 +40,9 @@ const envVars: EnvVars = {
 	NODE_ENV: nodeEnv,
 	SENTRY_DSN: process.env.SENTRY_DSN,
 	PROXY: getProxyFromEnvironment(),
-	MAINTENANCE_MODE: BooleanEnum[process.env.MAINTENANCE_MODE] || BooleanEnum.false,
+	LAUNCHDARKLY_KEY: process.env.LAUNCHDARKLY_KEY,
```

Comment:
I stashed the LaunchDarkly keys as secrets for the `ddev`, `stg-west`, and `prod-west` environments so that it connects to the proper LaunchDarkly project for the given environment.